src/ui/suspense/util.ts

```diff
@@ -6,36 +6,36 @@ import { pauseIdCache } from "replay-next/src/suspense/PauseCache";
 import { isExecutionPointsWithinRange } from "replay-next/src/utils/time";
 import { ReplayClientInterface } from "shared/client/types";
 
-// returns undefined if the async parent pause doesn't exist
-// or null if it is not in a loaded region
+// returns false if the async parent pause doesn't exist
+// or true if it is not in a loaded region
 export function getAsyncParentPauseIdSuspense(
   replayClient: ReplayClientInterface,
   pauseId: PauseId,
   asyncIndex: number,
   focusWindow: TimeStampedPointRange | null
-): PauseId | null {
+): PauseId | boolean {
   while (asyncIndex > 0) {
     const frames = framesCache.read(replayClient, pauseId)!;
     if (!frames?.length) {
-      return null;
+      return false;
     }
 
     const steps = frameStepsCache.read(replayClient, pauseId, frames[frames.length - 1].frameId);
     if (!steps || steps.length === 0) {
-      return null;
+      return false;
     }
 
     const executionPoint = steps[0].point;
     if (
       focusWindow === null ||
       !isExecutionPointsWithinRange(executionPoint, focusWindow.begin.point, focusWindow.end.point)
     ) {
-      return null;
+      return true;
     }
```

Comment:
Returning `true` here if `focusWindow === null` seems funky?

I don't think `focusWindow` window is ever `null` (it's either the full recording or a subset) _but_ if it were null, that would be _the same as_ the full recording, wouldn't it? In which case, returning `true` here seems like it would always be misleading.